docs/js-api.md

```diff
@@ -37,7 +37,8 @@ Options:
 - `virtual` - if true, JSS will use VirtualRenderer.
 - `insertionPoint` - string value of a DOM comment node which marks the start of sheets or a rendered DOM node. Sheets rendered by this Jss instance are inserted after this point sequentially.
 
-**Note**: Each `jss.setup()` call will overwrite all previous settings.
+**Note**: Each `jss.setup()` call will perform a shallow merge with the old options except for `plugins`. Passed `plugins` will be added to the existing plugins if they do not already exist.
```

Comment:
We have no duplicate detection right now

Reply:
It does call `#use` which does
https://github.com/cssinjs/jss/blob/dd20297caba69ac4c390f1460dca0034691f4a2a/packages/jss/src/Jss.js#L131

This seems like duplicate detection.